File: AtlasWeb/Core/Modules/Tibbi_Surec/Fizyoterapi_Modulu/PostureAnalysisFormViewModel.ts
//$0D062540
import { BaseViewModel } from "NebulaClient/Model/BaseViewModel";
import { PostureAnalysisForm } from "NebulaClient/Model/AtlasClientModel";
import { Type } from 'NebulaClient/ClassTransformer';

export class PostureAnalysisFormViewModel extends BaseViewModel {
    protected __type__: string = 'Core.Models.PostureAnalysisFormViewModel, Core';
    @Type(() => PostureAnalysisForm)

    public _PostureAnalysisForm: PostureAnalysisForm = new PostureAnalysisForm();
}

File: AtlasWeb/Core/Modules/Tibbi_Surec/Fizyoterapi_Modulu/PostureAnalysisFormViewModel.cs
//$D941E902
using System;
using System.Linq;
using Core.Models;
using Infrastructure.Filters;
using Infrastructure.Models;
using TTInstanceManagement;
using TTObjectClasses;
using TTDefinitionManagement;
using TTUtils;
using Core.Controllers;

namespace Core.Controllers
{
    public partial class PostureAnalysisFormServiceController
    {
    }
}

namespace Core.Models
{
    public partial class PostureAnalysisFormViewModel : BaseAdditionalInfoFormViewModel
    {
        public override BaseAdditionalInfo AddViewModelToContext(TTObjectContext objectContext)
        {
            var postureAnalysisFormService = new PostureAnalysisFormServiceController();
            postureAnalysisFormService.PostureAnalysisFormInternal(this, objectContext);
            return (PostureAnalysisForm)objectContext.GetObject(this._PostureAnalysisForm.ObjectID, "PostureAnalysisForm");
        }
    }
}
